tests/route/test_route_bgp_ecmp.py

```diff
@@ -0,0 +1,108 @@
+import requests
+import json
+import logging
+import time
+import pytest
+
+pytestmark = [
+    pytest.mark.topology("t0"),
+    pytest.mark.device_type('vs')
```

Comment:
Is this script specific for kvm? If not, I think this mark is unnecessary

Reply:
Will remove 'vs' mark.